CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
+- Fix: check endpoint expression when execute http command
```

Comment:
No issue associated to this PR? (just to check...)

Reply:
No issue was created